snet_cli/arguments.py

```diff
@@ -249,6 +280,81 @@ def add_contract_options(parser):
         add_contract_function_options(contract_p, contract_name)
 
 
+def _add_service_publish_arguments(parser):
+    parser.add_argument("--no-register", action="store_true", help="does not register the published service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
+    add_transaction_arguments(parser)
+
+
+def _add_service_update_arguments(parser):
+    parser.set_defaults(fn="update")
+    parser.add_argument("--new-price", help="new price to call the service", type=type_converter("uint256"))
+    parser.add_argument("--new-endpoint", metavar="ENDPOINT", help="new URL to call to reach the service")
+    parser.add_argument("--new-description", metavar="DESCRIPTION", help="new description for the service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
```

Comment:
metavar doesn't match the documentation

for *all* of the comments referring to metavar consistency, remember that metavar for "--" arguments (not positional arguments) defaults to caps and underscores (which is what you want most of the time)

e.g. `--name` metavar defaults to NAME
`--new-description` metavar defaults to NEW_DESCRIPTION
etc

So, in my opinion, it makes the most sense to avoid declaring a metavar on any of these unless you're really trying to override the default (which you aren't in most of these cases)

Reply:
Oh now I get it; removed everywhere; I changed the METAVAR for the "networks" subparser in publish and update to "[NETWORK]" to reflect that it's actually optional